app/models/user.js

```diff
@@ -0,0 +1,15 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'users',
+    {
+      firstName: { type: DataTypes.STRING, allowNull: false, field: 'first_name' },
```

Comment:
Define the id here, in order to get it when querying an User.

Reply:
Done.